models/netsuite/netsuite__balance_sheet.sql

```diff
@@ -30,6 +30,8 @@ subsidiaries as (
 
 balance_sheet as ( 
   select
+    transactions_with_converted_amounts.transaction_id,
+    transactions_with_converted_amounts.transaction_line_id,
```

Comment:
Is there any chance this could cause a breaking change? In particular I am wondering if there is a case where a customer passed these fields through via the `balance_sheet_transaction_detail_columns` variable?

Reply:
Same comment for the Income Statement model

Reply:
For example see the screenshot.

![image](https://user-images.githubusercontent.com/74217849/231855546-2653d378-70ff-4239-8d3d-5c9ea89df7c2.png)

Reply:
Although this is pretty minor, I do still worry this could be breaking for a number of customers. I feel the safe route would be to mark this as a breaking change and index the package version appropriately. What are your thoughts?

Reply:
ah good point, this is definitely a breaking change in that way. I guess it's only a breaking change for people using netsuite 1 though, right?

Reply:
Yeah you are right that it would _technically_ only be breaking for Netsuite1. However, we can't really separate out breaking changes for one and not the other. Or can we?

Would we be able to mark it as breaking just to be safe, but we can discuss as a team offline how we should handle situations like this in the future.

Reply:
No we can't separate them out, but I'll note that it's a breaking change in reality for Netsuite.com endpoint users